arviz/tests/test_data.py

```diff
@@ -548,6 +548,24 @@ def test_bad_inference_data():
         InferenceData(posterior=[1, 2, 3])
 
 
+def test_inference_concat_keeps_all_fields():

```

Comment:
I hope I am not too picky, it would be great to move the function below the other functions testing concat behaviour , around line [303](https://github.com/arviz-devs/arviz/blob/66bc210aa68efff1bb43a0821c6232826aaa90a6/arviz/tests/test_data.py#L303)

Reply:
Not a problem.